mantidproject/mantid
# LAGRANGE: Add instrument definition

which is used only to allow for distinction between data spectrum and the monitor one.

**Description of work.**

This PR adds a mockup instrument for Lagrange instrument. The dimensions or positions of the detector is not relevant in the reduction workflow. However, a way to properly distinguish data from monitor spectra requires an instrument, as well as to profit from parameters that can be stored in an Instrument Parameter File.

**To test:**
1. In workbench, run `LoadEmptyInstrument` with `InstrumentName` set to Lagrange.
2. Show data, it should have two spectra, with the second being a monitor.



Part of #35152  


*This does not require release notes* because ** the full release note of adding Lagrange will be added when the NeXus loader is implemented and tested.




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
